pkgs/top-level/all-packages.nix

```diff
@@ -17335,7 +17335,11 @@ with pkgs;
 
   qemu-riscv = callPackage ../applications/virtualization/qemu/riscv.nix {};
 
-  qgis = callPackage ../applications/gis/qgis {};
+  qgis-unwrapped = libsForQt5.callPackage ../applications/gis/qgis {
```

Comment:
Why expose the unwrapped version? Is it used somewhere?

Reply:
Exposing the unwrapped version in these cases is easiest way (IMO) to enable an overlay changing the source of the application. EG:

* https://github.com/coreyoconnor/nix_configs/commit/c37c85afa312e8ba79f9b96cbd4f49add62a9b56